awpy/types.py

```diff
@@ -665,14 +665,81 @@ class RoundStatistics(TypedDict):
     players_killed: dict[Literal["CT", "T"], set[str]]
 
 
-class FrameMapControl(TypedDict):
-    """Type for the result of awpy.analytics.map_control.calc_map_control.
+# Type to represent tile id for navigation tiles.
+# TODO: Incorporate TileIdObject into rest of repo.
+TileIdObject = NewType("TileIdObject", int)
```

Comment:
Yeah TileId sounds good to me.

Yeah there is one instance I handle a couple TileIds at once (`graph_to_neighbors` specifically) so maybe the NewType will be ok? I also was thinking it would be good to incorporate TileId into the rest of the package (nav package specifically) in the future which made me feel NewType would be ok

Reply:
I still feel that a TypeAlias suffices. Because in `graph_to_tile_neighbors` and the calling functions you are only handling TileIds anyway. A NewType adds the barrier that any external use needs to wrap it in the `TileId()`. This can be very helpful when you have a function or multiple closely related functions that all take integers/ids but where they come from different sources and could accidentally be swapped (like the example in the blog post). I dont feel this is the case here and so i think the disadvantage outweighs the benefit here.

We should still use a TypeAlias instead of just passing integers to make the code more readable but i feel the new type is overkill.

But we should still do a cleanup of the other nav and plot functions and incorporate the TypeAlias(ses) there.

Reply:
Yup changed to TypeAlias in next commit to branch.

Created an issue [here](https://github.com/pnxenopoulos/awpy/issues/240) to incorporate TileId into the rest of the repo.